src/modules/user/ordersHistory/reducer.ts

```diff
@@ -49,25 +50,25 @@ export const ordersHistoryReducer = (
         case ORDERS_HISTORY_DATA:
             return {
                 ...state,
-                list: action.payload.list,
+                list: action.payload.list.slice(0, defaultStorageLimit()),
                 fetching: false,
                 pageIndex: action.payload.pageIndex,
                 nextPageExists: action.payload.nextPageExists,
             };
         case ORDERS_HISTORY_RANGER_DATA:
-            return { ...state, cancelFetching: false, list: insertOrUpdate(state.list, convertOrderEvent(action.payload)) };
+            return { ...state, cancelFetching: false, list: insertOrUpdate(state.list, convertOrderEvent(action.payload)).slice(0, defaultStorageLimit()) };
         case ORDERS_HISTORY_ERROR:
             return { ...state, list: [], pageIndex: 0, fetching: false };
         case ORDERS_CANCEL_ALL_FETCH:
             return { ...state, cancelAllFetching: true, cancelAllError: false };
         case ORDERS_CANCEL_ALL_DATA:
-            return { ...state, cancelAllFetching: false, list: action.payload };
+            return { ...state, cancelAllFetching: false, list: action.payload.slice(0, defaultStorageLimit()) };
         case ORDERS_CANCEL_ALL_ERROR:
             return { ...state, cancelAllFetching: false, cancelAllError: true };
         case ORDERS_HISTORY_CANCEL_FETCH:
             return { ...state, cancelFetching: true, cancelError: false };
         case ORDERS_HISTORY_CANCEL_DATA:
-            return { ...state, cancelFetching: false, list: action.payload };
+            return { ...state, cancelFetching: false, list: action.payload.slice(0, defaultStorageLimit()) };
```

Comment:
In case of action.payload null, an error will be thrown.

I would suggest to write a helper for this, checking 
`function sliceArray(data, index) {
   if (!data || data.length === 0) {
       return [];
    }
    return data.slice(index)
}`